data/na/swagger/1-0-5.yaml

```diff
@@ -978,6 +978,9 @@ definitions:
       apple_pay_merchant_id:
         type: string
         description: The Apple Pay merchant ID.
+      passthrough:
```

Comment:
- passthrough should be first.
- Instead of saying "the transaction is externally decrypted" lets say "indicates that the transaction is externally decrypted".
- Mention that passthrough when false or null indicates a regular (Worldline decrypted) Apple Pay transaction.
- For apple_pay_merchant_id mention (maybe in brackets) that it is not required for passthrough.
- Update the description for paymentToken.
- Don't forget full stops.